packages/pwa-kit-runtime/src/ssr/server/extensibility/types.ts

```diff
@@ -6,21 +6,4 @@
  */
 import {Application} from 'express'
 
-/**
- * Interface that must be implemented by all Express application extensions.
- */
-export interface IExpressApplicationExtension {
-    /**
-     * The name of the extension that will be used for logging.
-     */
-    getName(): string
-
-    /**
-     * Called to enhance the Express application by adding routes, middleware, or any other functionalities.
-     * The method is predominantly used during server setup.
-     *
-     * @param app - The Express application instance to be extended.
-     * @returns The extended Express application instance.
-     */
-    extendApp(app: Application): Application
-}
+export type {Application}
```

Comment:
Minor: this type is no longer used. For example, see the other file 'application-extension.ts': https://github.com/SalesforceCommerceCloud/pwa-kit/blob/f1bc4aa9eae48d690b777e2dfacdfe8ccc8d28b8/packages/pwa-kit-runtime/src/ssr/server/extensibility/application-extension.ts#L7-L8

Reply:
Ahh yeah.. So actually I intended to use a re-exported type from the types file. The reason is that I think there are going to be more types in the future and I want to have all the type definitions together. This includes reexporting the express application in the case we ever extend or change the application type. 

If we find that we aren't requiring the type file and there is the only one re-export we can remove it.